src/config.js

```diff
@@ -10,12 +10,18 @@ if (dotenvResult.error) {
 
 // this is the message that will be signed by MetaMask and checked against when
 //  authenticating users
-const personalMessageToSign = 'Please sign this message to authenticate with the Codex Title Registry.'
+const typedDataToSign = [
+  {
+    type: 'string',
+    name: 'Sign In',
+    value: 'Sign in to Codex Title Viewer',
```

Comment:
Can we keep this message similar to what it was before?

```
Please sign this message to authenticate with the Codex Title Viewer.
```

Or is that too verbose 🤔